.github/workflows/ci.yml

```diff
@@ -14,8 +14,11 @@ jobs:
       image: quay.io/prometheus/golang-builder:1.22-base
     steps:
       - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+      - uses: actions/setup-go@0c52d547c9bc32b1aa3301fd7a9cb496313a4491 # v5.0.0
```

Comment:
```suggestion
      - uses: actions/setup-go@0c52d547c9bc32b1aa3301fd7a9cb496313a4491 # v5.0.0
         with:
           go-version: 1.22.x

```

Reply:
What do you think that will do?
I was relying on the documentation which said if there is already Go installed it will use that.

Reply:
Yes, but I think we want to make sure that we're specific here.